package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hubspot/local-dev-lib",
-  "version": "3.1.2",
+  "version": "0.1.2-experimental.0",
```

Comment:
This seems unintentional.

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/228/commits/460154dc57bf41d48bb20e6ba91f3f27bfed83ad